lib/node_modules/@stdlib/math/base/special/fast/alpha-max-plus-beta-min/lib/native.js

```diff
@@ -0,0 +1,46 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Computes the hypotenuse using the alpha max plus beta min algorithm.
+*
+* @param {number} x - number
```

Comment:
```suggestion
* @private
* @param {number} x - number
```

Native interfaces, such as in this package, are not exposed as public APIs. They are only for testing an implementation.

Reply:
added @private as this is an internal function to test implementation